coala_quickstart/generation/FileGlobs.py

```diff
@@ -28,6 +30,12 @@ def get_project_files(log_printer, printer, project_dir):
                       color="green")
         ignore_globs = get_gitignore_glob(project_dir)
 
+    if noninteractive:
+        if ignore_globs is None:
+            file = open(".gitignore", "w")
+            file.write("src")
+            ignore_globs = get_gitignore_glob(project_dir)
+
     if ignore_globs is None:
```

Comment:
Actually, the best way to solve this is to change this if as:

```
if ignore_globs is None:
    ...
elif not non_interactive:
    ignore_globs = []  # since ignore_globs is basically a list in the end, let's create an empty list
```

That way we don't even need to create an empty gitignore.